lib/fizzy/capi.cpp

```diff
@@ -205,7 +205,7 @@ inline FizzyExternalFunction wrap(fizzy::ExternalFunction external_func)
 {
     static constexpr FizzyExternalFn c_function =
         [](void* host_ctx, FizzyInstance* instance, const FizzyValue* args,
-            FizzyExecutionContext* c_ctx) -> FizzyExecutionResult {
+            FizzyExecutionContext* c_ctx) noexcept -> FizzyExecutionResult {
```

Comment:
It's a good change, but the function can throw `bad_alloc`.

Should it perhaps trap in case new context can't be allocated?

Reply:
Well the underlying host function API in C++ is noexcept, so it can't do much.

Not sure if trap or `std::terminate` is nicer :)

The latter may help the programmer to find the issue more easily?

Reply:
Should really try to make these functions dynamic allocation free.